gtests/net/mptcp/dss/dss_fin_retrans_established.pkt

```diff
@@ -0,0 +1,23 @@
+// Test data_fin retransmission from kernel TCP_ESTABLISHED state
+--tolerance_usecs=100000
+`../common/defaults.sh`
+
++0     socket(..., SOCK_STREAM, IPPROTO_MPTCP) = 3
++0     setsockopt(3, SOL_SOCKET, SO_REUSEADDR, [1], 4) = 0
+
++0     bind(3, ..., ...) = 0
++0     listen(3, 1) = 0
++0       <  S   0:0(0)         win 32792  <mss 1000, sackOK, nop, nop, nop, wscale 7, mpcapable v1 flags[flag_h] nokey>
++0       >  S.  0:0(0)  ack 1             <mss 1460, nop, nop, sackOK, nop, wscale 8, mpcapable v1 flags[flag_h] key[skey]>
++0.01    <   .  1:1(0)  ack 1  win 257                                               <mpcapable v1 flags[flag_h] key[ckey=2, skey]>
++0     accept(3, ..., ...) = 4
+
+// server shutdown
++0     close(4) = 0
++0       >   .  1:1(0)  ack 1             <dss dack4=1 dsn8=1 ssn=0 dll=1 nocs fin, nop, nop>
+
+// wait for retransmissions
++0.200   >   .  1:1(0)  ack 1             <dss dack4=1 dsn8=1 ssn=0 dll=1 nocs fin, nop, nop>
++0.200   >   .  1:1(0)  ack 1             <dss dack4=1 dsn8=1 ssn=0 dll=1 nocs fin, nop, nop>
++0.400   >   .  1:1(0)  ack 1             <dss dack4=1 dsn8=1 ssn=0 dll=1 nocs fin, nop, nop>
++0       <   .  2:2(0)  ack 1  win 450    <dss dack4=2 nocs>
```

Comment:
do we not have FIN in this case?

Reply:
There should be, yes. I was initially focused on verifying only the DATA_FIN retransmission part, but am definitely open to improving the tests.

Reply:
Just added :)

https://github.com/multipath-tcp/packetdrill/compare/96f908dcb7fb6c811d6baa449d9c9ca115acc56a..646649d925fdc16f3f399e695ed7a7c7df71f4c7

Is it OK for you?